sdk/ml/azure-ai-ml/azure/ai/ml/entities/_component/datatransfer_component.py

```diff
@@ -219,7 +218,6 @@ def __init__(
         outputs: Optional[Dict] = None,
         **kwargs,
     ):
-
```

Comment:
It looks like a linter went rogue here. Can you update only the files that you're touching?

Reply:
These changes happened after running the [black reformat command](https://msdata.visualstudio.com/Vienna/_git/sdk-cli-v2?path=/docs/azure-sdk-for-python/faq.md&version=GBmain&_a=preview&anchor=running-black) because that build gate was failing.